Libplanet.Tools/Mpt.cs

```diff
@@ -166,6 +167,68 @@ public class Mpt
             configurationService.Store(configuration);
         }
 
+        [Command(Description="List all aliases stored.")]
+        public void List(
+            [FromService] IConfigurationService<ToolConfiguration> configurationService)
+        {
+            ToolConfiguration configuration = configurationService.Load();
+            Dictionary<string, string> aliases = configuration.Mpt.Aliases;
+
+            int maxAliasLength = aliases.Keys.Max(alias => alias.Length),
+                maxPathLength = aliases.Values.Max(alias => alias.Length);
+            Console.Error.WriteLine(
+                $"{{0,-{maxAliasLength}}} | {{1,-{maxPathLength}}}",
+                "ALIAS",
+                "PATH");
+            Console.Error.WriteLine(new string('-', 3 + maxAliasLength + maxPathLength));
+            foreach (KeyValuePair<string, string> pair in aliases)
+            {
+                Console.Write($"{{0,-{maxAliasLength}}} ", pair.Key);
+                Console.Error.Write("| ");
+                Console.WriteLine($"{{0,-{maxPathLength}}}", pair.Value);
+            }
+        }
+
+        [Command(Description="Query a state of the state key at the state root hash.  It will " +
+                             "print the state as hexadecimal bytes string into stdout.  " +
+                             "If it didn't exist, it will not print anything.")]
```

Comment:
```suggestion
                             "If it isn't exist, it will not print anything.")]
```

Reply:
IMHO it's more natural for me.

```suggestion
                             "If it doesn't exist, it will not print anything.")]
```

Reply:
I amended it in 49a892d7c7fb5089a681bf26876b7737dd3474b1 🙏